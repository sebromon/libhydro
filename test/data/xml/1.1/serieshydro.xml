<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T10:05:00+03:00</DateHeureCreationFichier>
    <RefFichier>20100226160756_donnees_sac30-2010-02-26-15-05.xml</RefFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1520</CdIntervenant>
      <NomIntervenant>Service de Prévision des Crues (S.P.C.) Grand Delta</NomIntervenant>
      <CdContact>26</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <Series>
      <!-- serie 0 -->
      <Serie>
        <CdSiteHydro>V7144010</CdSiteHydro>
        <GrdSerie>Q</GrdSerie>
        <DtDebSerie>2010-02-26T13:10:00+02:00</DtDebSerie>
        <DtFinSerie>2010-02-26T14:55:00+02:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:10:00+02:00</DtObsHydro>
            <ResObsHydro>20992.0</ResObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:15:00+02:00</DtObsHydro>
            <ResObsHydro>21176.0</ResObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
      <!-- serie 1 -->
      <Serie>
        <CdStationHydro>V714401001</CdStationHydro>
        <GrdSerie>Q</GrdSerie>
        <DtDebSerie>2010-02-26T13:10:00</DtDebSerie>
        <DtFinSerie>2010-02-26T14:55:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <SysAltiSerie>0</SysAltiSerie>
        <SeriePerim>1</SeriePerim>
        <CdContact>233</CdContact>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:10:00</DtObsHydro>
            <ResObsHydro>20</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>12</QualifObsHydro>
            <ContObsHydro>false</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:15:00</DtObsHydro>
            <ResObsHydro>21</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>8</QualifObsHydro>
            <ContObsHydro>false</ContObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
      <!-- serie 2 -->
      <Serie>
        <CdCapteur>V71440100103</CdCapteur>
        <GrdSerie>H</GrdSerie>
        <DtDebSerie>2010-02-26T13:10:00</DtDebSerie>
        <DtFinSerie>2010-02-26T14:55:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <SysAltiSerie>31</SysAltiSerie>
        <SeriePerim>0</SeriePerim>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:10:00</DtObsHydro>
            <ResObsHydro>680.0</ResObsHydro>
            <MethObsHydro>4</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:15:00</DtObsHydro>
            <ResObsHydro>684.0</ResObsHydro>
            <MethObsHydro>0</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T14:55:00</DtObsHydro>
            <ResObsHydro>670.0</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
    </Series>
  </Donnees>
</hydrometrie>
