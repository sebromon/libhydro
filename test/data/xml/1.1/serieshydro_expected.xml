<?xml version="1.0" encoding="UTF-8"?>
<hydrometrie xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd">
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T07:05:00</DateHeureCreationFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1520</CdIntervenant>
      <NomIntervenant>Service de Prévision des Crues (S.P.C.) Grand Delta</NomIntervenant>
      <CdContact>26</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <Series>
      <Serie>
        <CdSiteHydro>V7144010</CdSiteHydro>
        <GrdSerie>Q</GrdSerie>
        <DtDebSerie>2010-02-26T11:10:00</DtDebSerie>
        <DtFinSerie>2010-02-26T12:55:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <SysAltiSerie>31</SysAltiSerie>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T11:10:00</DtObsHydro>
            <ResObsHydro>20992.0</ResObsHydro>
            <MethObsHydro>0</MethObsHydro>
            <QualifObsHydro>16</QualifObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T11:15:00</DtObsHydro>
            <ResObsHydro>21176.0</ResObsHydro>
            <MethObsHydro>0</MethObsHydro>
            <QualifObsHydro>16</QualifObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
      <Serie>
        <CdStationHydro>V714401001</CdStationHydro>
        <GrdSerie>Q</GrdSerie>
        <DtDebSerie>2010-02-26T13:10:00</DtDebSerie>
        <DtFinSerie>2010-02-26T14:55:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <SysAltiSerie>0</SysAltiSerie>
        <SeriePerim>true</SeriePerim>
        <CdContact>233</CdContact>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:10:00</DtObsHydro>
            <ResObsHydro>20.0</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>12</QualifObsHydro>
            <ContObsHydro>false</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:15:00</DtObsHydro>
            <ResObsHydro>21.0</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>8</QualifObsHydro>
            <ContObsHydro>false</ContObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
      <Serie>
        <CdCapteur>V71440100103</CdCapteur>
        <GrdSerie>H</GrdSerie>
        <DtDebSerie>2010-02-26T13:10:00</DtDebSerie>
        <DtFinSerie>2010-02-26T14:55:00</DtFinSerie>
        <StatutSerie>4</StatutSerie>
        <DtProdSerie>2010-02-26T14:55:00</DtProdSerie>
        <SysAltiSerie>31</SysAltiSerie>
        <SeriePerim>false</SeriePerim>
        <ObssHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:10:00</DtObsHydro>
            <ResObsHydro>680.0</ResObsHydro>
            <MethObsHydro>4</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T13:15:00</DtObsHydro>
            <ResObsHydro>684.0</ResObsHydro>
            <MethObsHydro>0</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
          <ObsHydro>
            <DtObsHydro>2010-02-26T14:55:00</DtObsHydro>
            <ResObsHydro>670.0</ResObsHydro>
            <MethObsHydro>12</MethObsHydro>
            <QualifObsHydro>20</QualifObsHydro>
            <ContObsHydro>true</ContObsHydro>
          </ObsHydro>
        </ObssHydro>
      </Serie>
    </Series>
  </Donnees>
</hydrometrie>
